.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
+charset = utf-8
```

Comment:
for files upload test cases we use ASCII encoding, else the test cases uploading files will be converted to UTF-8 and will cause test case failures.

Reply:
ASCII is a subset of UTF-8 so I don't think it would matter!  VS Code also defaults to UTF-8 so its likely that this wouldn't impact anyone but it does ensure that if you have customized it that while editing files in the SIMS root it will be UTF-8.

Reply:
@guru-aot the settings are following what we have already using and they will not affect the existing files.
For instance, with the settings in place, opening a ts file and saving it would not cause any difference.
The ASCII files will not be affected.
@sslaws, yes, the ASCII would not matter. The problem that @guru-aot was trying to make was about the extended ASCII (8bits) that has issues while converting to UTF-8, but either way would not be a problem and we were not even sure at this moment that we need to support extended ASCII.